README.md

```diff
@@ -1 +1,60 @@
-# Javascript-Capstone
\ No newline at end of file
+# Javascript-Capstone
+
+## Description
+
+- This project is about building a site for Pokemons lovers that shows a number of pokemons and some details on each
+- The second part is creating a pop up windows that contains the photo of the pokemon, its abilities, specie etc...
+- One of the features is to have a like button for the user to demonstrate his likeness of the pokemon and also a like counter below it that saves the user's likes to the envolvement API.
+- The other prominent feature is the comment section that allows the user to post a comment on his favourite Pokemon and it displays the previous comments posted by other users also
+- The page also has a header that contains the slogan and the pokemon counter that gives an exact count of pokemons on screen
+- At the bottom of the page there is the footer that conatins the license and the develpoers signature.
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- Node-modules
+- Lighthouse
+- Hintweb
+- Stylint
+- Webpack
+- Jest
+- NPM
+- Web server
+
+## Links
+
+  this is the link to the page on internet.
+
+## Authors
+
+👤 **Bassem Shams**
+
+- GitHub: [@githubhandle](https://github.com/basem909)
+
+👤 **Rafael Guzman**
+
+- GitHub: [@githubhandle](https://github.com/rega1237)
+
+
+## Loom video link
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is MIT licensed.
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- copy this link `https://github.com/rega1237/Javascript-Capstone.git`.
+- get the directory that you want to clone the repository.
+- open the command prompt in this directory.
+- write `git clone https://github.com/rega1237/Javascript-Capstone.git`.
+- go to the repository folder in your command prompt, run `cd Javascript-Capstone`.
+- run `npm install`
```

Comment:
- [ ] Please move this section straight below the "Built with" section because it looks more professional as suggested by Microverse [Readme template](https://github.com/microverseinc/readme-template/blob/master/README.md). 
- [ ] I suggest you remove the unused sections and re-arrange your sections following the Microverse template([See ReadMe](https://github.com/microverseinc/readme-template/blob/master/README.md)), also do not forget to send your video link to Readme.